client-ui/src/app/generated/models.ts

```diff
@@ -0,0 +1,14 @@
+export { Attribute } from './models/attribute';
+export { DataSourceInfo } from './models/data-source-info';
+export { DataType } from './models/data-type';
+export { ExecutionEventInfo } from './models/execution-event-info';
+export { ExecutionPlanInfo } from './models/execution-plan-info';
+export { LineageDetailed } from './models/lineage-detailed';
+export { LineageDetailedGraph } from './models/lineage-detailed-graph';
+export { LineageOverview } from './models/lineage-overview';
+export { LineageOverviewGraph } from './models/lineage-overview-graph';
+export { LineageOverviewNode } from './models/lineage-overview-node';
+export { Operation } from './models/operation';
+export { OperationDetails } from './models/operation-details';
+export { PageableExecutionEventsResponse } from './models/pageable-execution-events-response';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [' should be "](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4584725)